src/main/scala/com/amazon/deequ/profiles/ColumnProfilerRunBuilder.scala

```diff
@@ -40,6 +40,8 @@ class ColumnProfilerRunBuilder(val data: DataFrame) {
   protected var saveConstraintSuggestionsJsonPath: Option[String] = None
   protected var saveEvaluationResultsJsonPath: Option[String] = None
   protected var kllParameters: Option[KLLParameters] = None
+  protected var predefinedColumnDataTypes: Map[String, DataTypeInstances.Value] =
```

Comment:
please rename this to predefinedTypes (everywhere) and change the default argument to Map.empty

Reply:
Ok!